lib/wda/utils.js

```diff
@@ -156,6 +157,47 @@ CODE_SIGN_IDENTITY = ${signingId}
   return xcconfigPath;
 }
 
+/**
+ * Creates xctestrun file per device & platform version.
+ * We expects to have WebDriverAgentRunner_iphoneos${platformVersion}-arm64.xctestrun for real device
+ * and WebDriverAgentRunner_iphonesimulator${platformVersion}-x86_64.xctestrun for simulator located @bootstrapPath
+ *
+ * @param {boolean} isRealDevice - Equals to true if the current device is a real device
+ * @param {string} udid - The device UDID.
+ * @param {string} platformVersion - The platform version of OS.
+ * @param {string} bootstrapPath - The device UDID.
+ * @param {string} wdaRemotePort - The remote port WDA is listening on.
```

Comment:
please add _return_ and _throws_ descriptions

Reply:
done